main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsFareAttributeSchema.java

```diff
@@ -0,0 +1,32 @@
+package org.mobilitydata.gtfsvalidator.table;
+
+import org.mobilitydata.gtfsvalidator.annotation.*;
+
+import java.util.Currency;
+
+@GtfsTable("fare_attributes.txt")
+public interface GtfsFareAttributeSchema extends GtfsEntity {
+    @FieldType(FieldTypeEnum.ID)
+    @PrimaryKey
+    @Required
+    String fareId();
+
+    @NonNegative
+    double price();
+
+    @Required
+    Currency currencyType();
```

Comment:
Should this be annotated as "@FieldType(FieldTypeEnum.CURRENCY_CODE)"?

Reply:
No because it is already a `Currency`.